h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -412,6 +412,9 @@ def _metrics_class(model_json):
 
     def convert_H2OXGBoostParams_2_XGBoostParams(self):
         '''
+        This method and convert_H2OFrame_2_DMatrix will require to import the following toolboxes:
+        xgboost, pandas, numpy and scipy.sparse.
+
```

Comment:
In order to use convert_H2OXGBoostParams_2_XGBoostParams and convert_H2OFrame_2_DMatrix, you must import the following toolboxes:

Reply:
Thanks.  Done